src/scripts/main.js

```diff
@@ -21,7 +21,21 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const topUL = document.createElement('ul');
+
+  for (const topKey in data) {
+    const topLI = document.createElement('li');
+
+    topLI.textContent = topKey;
+
+    if (Object.keys(data[topKey]).length > 0) {
```

Comment:
The condition `Object.keys(data[topKey]).length > 0` checks if there are any children to process. This is correct, but consider using `if (data[topKey] && Object.keys(data[topKey]).length > 0)` to ensure 'data[topKey]' is not null or undefined before accessing its keys.